tests/Feature/AdjustmentTest.php

```diff
@@ -32,6 +34,31 @@ public function test_can_adjust_wallet_balance($initial_balance, $new_balance)
                 'id' => 1,
                 'balance' => $new_balance,
             ]);
+
+        $this->assertActionUsesFormRequest(
+            WalletAdjustmentController::class,
+            'balance',
+            WalletAdjustmentRequest::class
+        );
```

Comment:
We need to check this once, I think the best way is to have a Unit test file per FormRequest that contains two tests about the Rules array and the Action uses form request 

``` php
public function testRules()
    {
        $this->assertEquals([
                'name' => 'required',
                'email' => 'required|email'
            ],
            (new WalletAdjustmentController)->rules()
        );
    }
```

Reply:
I don't know if I get it right, but check out what I have done.